main/main.cpp

```diff
@@ -1182,6 +1183,26 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			OS::get_singleton()->disable_crash_handler();
 		} else if (I->get() == "--skip-breakpoints") {
 			skip_breakpoints = true;
+		} else if (I->get() == "--xr-mode") {
+			// long form switch allows us to set a specific mode
```

Comment:
No longer accurate comment.

Reply:
I removed it all together, it only made sense when we had the `-xr` switch which I've removed.